### DISCUSSION TITLE
Wallust to replace Pywal for color palette - branch

### DISCUSSION THREAD

**Discussion by JaKooLit** [JaKooLit/Hyprland-Dots#251] [Ideas]
Feel free to test.....

requirements: you need the dev branch of wallust
https://codeberg.org/explosion-mental/wallust/src/branch/dev

for Arch linux users, you can just install the wallust-git from AUR

the old versions of wallust will not work

https://github.com/JaKooLit/Hyprland-Dots/tree/development-wallust

Please remember, this is on development-wallust branch

to clone

git clone https://github.com/JaKooLit/Hyprland-Dots --depth 1  -b development-wallust

Again a reminder you need the wallust dev version NOT the stable version

**Comment by SherLock707** [comment]
Hey @JaKooLit , Did you consider [Matugen](https://github.com/InioX/matugen) ?

It has Light/Dark pallet support + same kind of template target feature and very nice colour variations: [https://github.com/InioX/matugen/wiki/Configuration](https://github.com/InioX/matugen/wiki/Configuration)

![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/fa2a8186-0e08-42ac-8231-6a02546820a5)


Wallust colour is bit on dull side by default?
 (Left Wallust) (Right Pywall)
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/2c6615dc-be06-47b5-b414-fabdc1a73c90)

**Comment by JaKooLit** [comment]
on wallust, which pallete and color_space you use?

**Reply by SherLock707** [reply]
Same ones as from your wallust config.

**Comment by JaKooLit** [comment]
there is also threshold for wallust

**Reply by SherLock707** [reply]
I'll check with different values.

**Reply by JaKooLit** [reply]
cool.. let me know whats your finding...

Im reluctant to move to wallust actually... development is very slow.. and then Pywal have been archived .... cant find from net best alternative...
